### PR TITLE
Adding Bootstrap-touchspin widget

### PR DESCRIPTION
**Pull request by rhamilto**
@andresgalante or @erundle, please review and merge.

### CODE REVIEW

File: less/bootstrap-touchspin.less [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//
+// Bootstrap Touchspin
+// --------------------------------------------------
+
+.bootstrap-touchspin {
+  .input-group-btn-vertical {
```
Comment (erundle) [inline less/bootstrap-touchspin.less]: So I know you have been writing selectors in this fashion but should we look at reducing the nesting of this to .bootstrap-touchspin .input-group-btn-vertical

File: less/patternfly-additions.less [hunk 2]
```diff
@@ -24,6 +26,7 @@
 @import "bootstrap-datepicker.less";
 @import "bootstrap-select.less";
 @import "bootstrap-switch.less";
+@import "bootstrap-touchspin.less";
```
Comment (erundle) [inline less/patternfly-additions.less:29]: This isn't related to this but maybe in the future we rename these to something with overrides or pf whatever so you can see from the source map where this coming from (our or theirs) easily

File: components/bootstrap-touchspin/demo/demo.css [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+}
+
+.navbar-default {
+    font-family: "Century Gothic", Arial, sans-serif;
+    background: #000 url(img/bg-menu.png) repeat-x left top; /*#377fa0;*/
+    color: #fff;
+    text-transform: uppercase;
+    border: none;
+    box-shadow: 0px 0px 1px #000;
+    /*background-image: linear-gradient(to top, rgb(62, 86, 112) 0%, rgb(69, 94, 122) 100%)*/;
```
Comment (andresgalante) [inline components/bootstrap-touchspin/demo/demo.css:101]: There is an extra `;` after the commented line

File: less/patternfly-additions.less (hunk 2 above)
Comment (andresgalante) [inline less/patternfly-additions.less:29]: :+1:

**Comment by andresgalante** [comment]
That `;` on the demo doesn't really matter. I am ok to merge this.

**Pull request merged**